app/views/devise/sessions/new.html.erb

```diff
@@ -34,11 +34,14 @@
       ) %>
   <%= f.input :request_id, as: :hidden, input_html: { value: @request_id } %>
   <div class='margin-bottom-4'>
-    <%= f.submit t('links.next'), full_width: true, big: false, wide: false, class: 'margin-bottom-2' %>
+    <%= f.submit t('links.next'), full_width: true, big: false, wide: false, class: 'margin-bottom-3' %>
+    <h2 class='margin-top-1 margin-bottom-3 text-normal text-center text-center--line text-bold border-bottom'>
```

Comment:
This has both `text-normal` and `text-bold` classes. Based on the Figma design, I'm assuming it should only be `text-bold`.